evolvedbinary/prosemirror-lwdita
# Remove token from local storage and store it as global variable

# The problem
The user token we got back from GitHub is stored in localstorage, this can pose a security risk.

# The fix
The user token will only be stored in memory

# How to test this
1. Visit http://localhost:1234/?ghrepo=evolvedbinary/cityehr-documentation&source=cityehr-quick-start-guide/src/main/lwdita/quickstart-guide-modular/verify-install.dita&branch=develop&referer=https://evolvedbinary.github.io/cityehr-documentation/verify-install.html
2. Clear application storage
3. Click on publish button and create a PR
4. Note the PR was created
